omim2obo/parsers/omim_txt_parser.py

```diff
@@ -224,8 +224,22 @@ def get_hgnc_map(filename, symbol_col, mim_col='MIM Number') -> Dict:
     """Get HGNC Map"""
     map = {}
     input_path = os.path.join(DATA_DIR, filename)
-    df = pd.read_csv(input_path, delimiter='\t', comment='#').fillna('')
-    df[mim_col] = df[mim_col].astype(int)  # these were being read as floats
+    try:
+        df = pd.read_csv(input_path, delimiter='\t', comment='#').fillna('')
+        df[mim_col] = df[mim_col].astype(int)  # these were being read as floats
+    # TODO: Need a better solution than this. Which should be: When these files are downloaded, should uncomment header
```

Comment:
## The issue
I'm not sure why this suddenly started happening, where the header is missing from the files. This is what is causing the KeyError causing the action to fail.

## The solution
The best way to fix this would be to do things at download time for all the files. But in the interest of time, I fixed it at the point of failure.
Need a better solution than this. Which should be: When these files are downloaded, should uncomment header

Reply:
What issue is this solving? Sorry I was in a big hole, but getting back to life now!

Reply:
That's all right. Thanks for checking in on this. I wanted more time to look more deeply into the root cause and fix the issue upstream in a better way, but basically I did a quick fix.

Basically what's happening is that when the TSVs are downloaded, there are a few comment lines, including the header is commented out. So we do a special operation to uncomment that header . For some reason it suddenly it wasn't getting uncommented